docs/snippets/idempotency/templates/tableCdk.ts

```diff
@@ -0,0 +1,28 @@
+import { Stack, type StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+import { AttributeType, BillingMode, Table } from 'aws-cdk-lib/aws-dynamodb';
+
+export class IdempotencyStack extends Stack {
+  public constructor(scope: Construct, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const table = new Table(this, 'idempotencyTable', {
+      partitionKey: {
+        name: 'id',
+        type: AttributeType.STRING,
+      },
+      timeToLiveAttribute: 'expiration',
+      billingMode: BillingMode.PAY_PER_REQUEST,
+    });
+
+    const fnHandler = new NodejsFunction(this, 'helloWorldFunction', {
```

Comment:
can we add `entry` and `handler` as a place holder? I understand that `NodejsFunction` can be smart for specific file and resource name convention. But it'd be great to be more explicit for the folks who are new.

Reply:
Good point, I'll add it later today